src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.tsx

```diff
@@ -4,6 +4,7 @@ import classNames from "classnames";
 import {REACTION_EMOJI_MAP, ReactionType} from "types/reaction";
 import {ReactionModeled} from "../NoteReactionList";
 import "./NoteReactionBar.scss";
+import {useAppSelector} from "../../../../store";
```

Comment:
just using "store" as import path is enough. also works for "types", "constants", "components" etc.  
Also always put the stylesheet as last item